ebb_lint/test/test_flake8.py

```diff
@@ -400,6 +400,30 @@ class C(object):
 $L204$'a' 'b'  $L303$# noqa
 
     ''',
+
+    # Oh boy these following two cases need explanation. The prefix of the
+    # 'spam' node in both of these cases ends with a literal \ followed by a
+    # newline, which cannot be tokenized by generate_tokens because it expects
+    # something to follow that sequence. This is one hell of a special case,
+    # but it made flake8 fail hard on valid syntax. The first case is a prefix
+    # containing no comment, and the second case also includes a comment which
+    # requires scanning.
```

Comment:
That's the line-continuation syntax, yeah? What's the particular part that's making it choke, the part where it's right after an assignment operator?